zips/zip-0246.rst

```diff
@@ -32,41 +32,58 @@ field, as specified in section 7.1 of the Zcash protocol specification
 Abstract
 ========
 
-This ZIP defines the digest algorithms associated with the v6 transaction format.
-
-This proposal also defines the new concept of "sighash versioning": where previously
-each transaction version had a single associated digest algorithm, going forward it
-will be possible for signers to use any digest algorithm within the closed set
-specified for a given transaction version (and made available in consensus via network
-upgrades).
+This ZIP defines the digest algorithms associated with the v6 transaction
+format.
 
+This proposal also defines the new concept of "sighash algorithm versioning":
+where previously each transaction version had a single associated digest
+algorithm, going forward it will be possible for signers to use any digest
```

Comment:
```suggestion
where previously each transaction version had a single associated sighash
algorithm, going forward it will be possible for signers to use any sighash
```